.github/workflows/generate-kubelet-flags.yaml

```diff
@@ -0,0 +1,48 @@
+name: Generate valid Kubelet Flags per k8s version
+on:
+  workflow_dispatch: {}
+
+jobs:
+  generate-kubelet-flags:
+    runs-on: ubuntu-latest
+    steps:
+    - name: Set up containerd
+      uses: crazy-max/ghaction-setup-containerd@v2
```

Comment:
nit: this feels like an easy one to do + match with our existing containerd version (but can be from upstream vs CDN)

https://github.com/alexeldeib/devbox/blob/main/devbox.sh for example

Reply:
ack, can take it up as a follow up, in favor of wrapping up the core functionality first, if that is ok